core/src/main/java/com/digitalpebble/storm/crawler/bolt/FetcherBolt.java

```diff
@@ -664,6 +671,9 @@ public void prepare(Map stormConf, TopologyContext context,
         this.metricGauge = context.registerMetric("fetcher",
                 new MultiCountMetric(), 10);
 
+        this.averagedMetrics = context.registerMetric("fetcher_reduced",
```

Comment:
Yeah that's fine with me. I doubt we'll be using any reducer other than `MeanReducer`, so that should be just as accurate